node-module-system/calculate.js

```diff
@@ -0,0 +1,28 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const [, , ...values] = process.argv;
+const [firstNumericValue, operation, secondNumericValue] = values;
+let result;
+switch (operation) {
+  case 'plus':
+    result = add(Number(firstNumericValue), Number(secondNumericValue));
+    console.log('result:', result);
+    break;
+  case 'minus':
+    result = subtract(Number(firstNumericValue), Number(secondNumericValue));
+    console.log('result:', result);
+    break;
+  case 'over':
+    result = divide(Number(firstNumericValue), Number(secondNumericValue));
+    console.log('result:', result);
+    break;
+  case 'times':
+    result = multiply(Number(firstNumericValue), Number(secondNumericValue));
```

Comment:
I have no objections to declaring `result` above as a `let`, but I thought I'd let you know that it is possible to do it without declaring a "global" result variable:
```
case 'times':
  {
    const result = multiply(firstNumericValue, secondNumericValue);
    console.log('result:', result);
   }
  break;
```
Which you use is a matter of taste.

And another alternative, which would make the switch shorter and more focused, is to move all those duplicate `console.log` statements to the bottom:

```
let result;
switch (operation) {
  case 'times':
    result = multiply(firstNumericValue, secondNumericValue);
    break;
....
}
console.log(result !== undefined ? `result: ${result}` : `invalid operation, ${operation}`;
```